src/SSHDebugPS/StringResources.resx

```diff
@@ -172,16 +172,32 @@
     <value>The specified directory {0} could not be created or accessed.</value>
     <comment>{0} directory name.</comment>
   </data>
+  <data name="Error_OperationTimedOut" xml:space="preserve">
+    <value>Operation timed out.</value>
+  </data>
+  <data name="Error_ProcessListFailedFormat" xml:space="preserve">
+    <value>{0}
+
+{1}</value>
+    <comment>{0} = 'ps' error message, {1} = procfs error message</comment>
+  </data>
+  <data name="Error_ProcessListFailedTitle" xml:space="preserve">
+    <value>Unable to obtain process list</value>
```

Comment:
>list [](start = 36, length = 4)

Add a period at the end?

Reply:
This is the title. Do you want a period in the title?

Reply:
@andrewcrawley convinced me why i should period it.